react-native-libraries.json

```diff
@@ -2751,6 +2760,7 @@
   },
   {
     "githubUrl": "https://github.com/expo/expo/tree/main/packages/expo-face-detector",
+    "examples": ["https://docs.expo.dev/versions/latest/sdk/facedetector/#example"],
     "ios": true,
```

Comment:
It seems like checks are failing for expo-face-detector...

Reply:
Ah because the library is no longer on expo/expo main. Let me remove this entry since it is already deprecated from SDK 52.